x/pki/keeper/msg_server_add_pki_revocation_distribution_point.go

```diff
@@ -63,6 +64,16 @@ func (k msgServer) AddPkiRevocationDistributionPoint(goCtx context.Context, msg
 		return nil, pkitypes.NewErrPkiRevocationDistributionPointAlreadyExists("PKI revocation distribution point already exist")
 	}
 
+	revocationList, isFound := k.GetPkiRevocationDistributionPointsByIssuerSubjectKeyID(ctx, msg.IssuerSubjectKeyID)
+	if isFound {
+		for _, revocationPoint := range revocationList.Points {
+			if revocationPoint.DataURL == msg.DataURL {
```

Comment:
what about equality of VID? I think it also needs to be checked, as we require uniqueness of (VendorID, IssuerSubjectKeyID, DataUrl) combination.